### ISSUE TITLE
go: GOPATH entry is relative; must be absolute path: "\\ext-go\\1".

### ISSUE THREAD

**Issue by x04** [techknowlogick/xgo#101]
I get this error when trying to use xgo:
```
go: GOPATH entry is relative; must be absolute path: "\\ext-go\\1".
For more details see: 'go help gopath'
```

On Windows 10 with go 1.16, I'm trying to build a local package inside a go module.

**Comment by techknowlogick** [comment]
Thanks for the report, could you provide the exact xgo command you used to trigger this error, and if possible a repo too?

**Comment by x04** [comment]
The command is 
```xgo --targets=darwin/amd64 ./cmd/main```

The repo is private, however all it is at the moment is a go module and that package starts a web server and navigates the web view to it.

**Comment by techknowlogick** [comment]
Are you running xgo on windows, or some other platform?

**Comment by fishersyu** [comment]
go: updating go.sum: open /ext-go/1/src/asrfile/go.sum: read-only file system
2021/02/27 20:55:28 Failed to cross compile package: exit status 1

On mac 10.15.7 with go 1.15.8

**Comment by x04** [comment]
> Are you running xgo on windows, or some other platform?

Windows

**Comment by vadipp** [comment]
I'm having the same problem. I'm running xgo on a Windows 10 system with Docker for Windows with WSL2 backend.

My command is `xgo --pkg=main --targets=linux/mipsle ./`.

**Comment by AbsoluteValentine** [comment]
you should add 'filepath.ToSlash()' to the following code
![image](https://user-images.githubusercontent.com/62891385/181197686-d7637553-7417-4529-96ab-a784dfdf0aae.png)

**Comment by zeripath** [comment]
Should be fixed by #165
